ReducingCrime.Rmd

```diff
@@ -85,7 +96,7 @@ summary(crimeData$prbpris)
 ## Univariate Variable Analysis of Key Variables
 
 ### crmrte: crimes committed per person
-Looking at the histogram of crime per person, the distribution appears to be positively skewed. Applying `log()` on crime shows the histogram to appear normally distributed.
+Looking at the histogram of crime per person, the distribution appears to be positively skewed. Applying `log()` on crime shows the histogram to appear normally distributed. The linear regressions would benefit from this transformation.
```

Comment:
Crime Rate is between 0 and 1, as Eric said in latest live class, log shouldn't be applied to very small values as log of small values tends to negative infinity and difficult to manage. What do you say?

Reply:
I think we convert to a higher scale say crime rate per million or something to bring sensible numbers out of log and easy to interpret the conclusion

Reply:
good point on the log. yes we should take this out. let us discuss the latter when we meet.